.gitignore

```diff
@@ -5,3 +5,4 @@ coverage
 vendor
 examples/**/dev.log
 examples/**/sessions
+.idea
```

Comment:
please revert this, it's specific to your setup

a common practice to add OS/IDE specific things to gitignore, is using a [global one](https://stackoverflow.com/questions/7335420/global-git-ignore)

Reply:
Brill, thanks for flagging will push an update